.circleci/config.yml

```diff
@@ -1087,22 +1087,6 @@ workflows:
                 - trunk
                 # Do not spin up calypso.live for fork pull requests. Calypso.live will not build them.
                 - /pull\/[0-9]+/
-      #      - test-e2e-canary:
-      #          requires:
-      #            - wait-calypso-live
-      #          test-flags: '-C -S $CIRCLE_SHA1'
-      # - test-e2e-canary:
-      #     name: test-e2e-canary-ie
-      #     requires:
-      #       - wait-calypso-live
-      #     test-flags: '-z -S $CIRCLE_SHA1'
-      #     test-target: 'IE11'
-      #     slack-webhook: '$SLACK_IE'
-      - test-e2e-canary:
-          name: test-e2e-canary-safari
-          requires:
-            - wait-calypso-live
```

Comment:
We can probably remove `wait-calypso-live` as well, I don't think anybody depends on it anymore.

Reply:
I've pushed a new revision with all references to `wait-calypso-live` removed.

`circleci config validate` reports no issues. Waiting for CI to complete its checks.